chat-agent/main.py

```
@@ -67,6 +142,61 @@ async def health():
+        - is_flagged: True if content violates policies
+        - category: The moderation category flagging the content (e.g., "hate", "sexual", "violence")
+    """
+    try:
+        response = await moderation_client.moderations.create(input=text)
+        result = response.results[0]
+        if result.flagged:
+            for category, flagged in result.categories.model_dump().items():
+                if flagged:
+                    return True, category
+        return False, ""
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)